src/components/TagSearch.tsx

```diff
@@ -114,7 +114,7 @@ export default class WithCallbacks extends React.Component<Props, State> {
 				defaultOptions={this.state.defaultOptions}
 				onInputChange={this.handleInputChange}
 				isMulti
-				placeholder="Search…"
+				placeholder="Search …"
```

Comment:
IMHO such things are commonly written without the space...

<img width="226" alt="Screenshot 2019-05-27 at 10 02 52" src="https://user-images.githubusercontent.com/316435/58404936-38383680-805e-11e9-9078-d1e08e669e66.png">